decentraland/unity-renderer
# style: controls screen update

## What does this PR change?
The key preview assets and the close button container have been updated. The builder key removed and the social keys reorganised that were misplaced. 

**Previous HUD**
<img width="1776" alt="Screenshot 2022-08-11 at 14 39 42" src="https://user-images.githubusercontent.com/51088292/204542344-375fa9e9-9f08-4f47-9136-4ac5e935d0fd.png">

**New HUD**
<img width="963" alt="Screenshot 2022-11-29 at 10 33 57" src="https://user-images.githubusercontent.com/51088292/204542614-5fd6d3ec-a281-410c-bd61-f2b6144f48a2.png">

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=style/controls-hud-update
2. Press the key [C] and compare with the previous version that now the keys are well placed and that the builder reference doesn't exist anymore.
3. Check that the functionality open close pressing [C] works fine. Press [Esc] and the close button to close the panel too.
